[2023-04-10T15:59:02.753+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgress_hooks_v099.postgres_to_s3 manual__2023-04-10T15:55:31.960398+00:00 [queued]>
[2023-04-10T15:59:02.764+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgress_hooks_v099.postgres_to_s3 manual__2023-04-10T15:55:31.960398+00:00 [queued]>
[2023-04-10T15:59:02.764+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-10T15:59:02.764+0000] {taskinstance.py:1289} INFO - Starting attempt 2 of 7
[2023-04-10T15:59:02.764+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-10T15:59:02.786+0000] {taskinstance.py:1309} INFO - Executing <Task(PythonOperator): postgres_to_s3> on 2023-04-10 15:55:31.960398+00:00
[2023-04-10T15:59:02.789+0000] {standard_task_runner.py:55} INFO - Started process 2258 to run task
[2023-04-10T15:59:02.790+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgress_hooks_v099', 'postgres_to_s3', 'manual__2023-04-10T15:55:31.960398+00:00', '--job-id', '7101', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgress_hooks.py', '--cfg-path', '/tmp/tmphhsri0gi']
[2023-04-10T15:59:02.791+0000] {standard_task_runner.py:83} INFO - Job 7101: Subtask postgres_to_s3
[2023-04-10T15:59:02.856+0000] {task_command.py:389} INFO - Running <TaskInstance: dag_with_postgress_hooks_v099.postgres_to_s3 manual__2023-04-10T15:55:31.960398+00:00 [running]> on host d034b084d02f
[2023-04-10T15:59:02.958+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=dejan
AIRFLOW_CTX_DAG_ID=dag_with_postgress_hooks_v099
AIRFLOW_CTX_TASK_ID=postgres_to_s3
AIRFLOW_CTX_EXECUTION_DATE=2023-04-10T15:55:31.960398+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-10T15:55:31.960398+00:00
[2023-04-10T15:59:02.961+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205 AirflowContextDeprecationWarning: Accessing 'next_ds_nodash' from the template is deprecated and will be removed in a future version. Please use '{{ data_interval_end | ds_nodash }}' instead.
[2023-04-10T15:59:02.970+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-04-10T15:59:02.974+0000] {dag_with_postgress_hooks.py:44} INFO - Saved orders data in text file: dags/get_orders_20230410.txt
[2023-04-10T15:59:02.974+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-04-10T15:59:02.989+0000] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=dag_with_postgress_hooks_v099, task_id=postgres_to_s3, execution_date=20230410T155531, start_date=20230410T155902, end_date=20230410T155902
[2023-04-10T15:59:03.010+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-04-10T15:59:03.032+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
